src/OpenSage.Game/Content/SubsystemLoader.cs

```diff
@@ -110,6 +110,7 @@ private IEnumerable<string> GetSubsystemEntryName(Subsystem subsystem)
                         case SageGame.Bfme:
                         case SageGame.Bfme2:
                         case SageGame.Bfme2Rotwk:
+                            LoadFiles(@"Maps\MapCache.ini");
```

Comment:
Could you move this logic to `GetFilesForSubsystem` or `Load` instead? It doesn't feel right that a side-effect free method like this one should open files and parse INIs.

Reply:
Yup, will do